environments/staging/terraform.yml

```diff
@@ -429,7 +429,7 @@ r53_private_zone:
 efs_file_systems:
   - efs_name: "efs_staging"
     create: yes
-    transition_to_ia:  "AFTER_14_DAYS"
+    transition_to_ia:  "AFTER_7_DAYS"
```

Comment:
Can you explain why this change was necessary for v12?

Reply:
This change does not relate to v12(sorry for using `In the process of v12 validation & testing` in the initial comment). In the general practice & to reduce EFS storage price, I am updating the attribute for moving the files which have not been accessed for the last 7 days to the IA storage class.